public/app/plugins/datasource/grafana/timeRegions.test.ts

```diff
@@ -0,0 +1,446 @@
+import { dateTime, toDataFrameDTO } from '@grafana/data';
+
+import { doTimeRegionQuery } from './timeRegions';
+
+describe('grafana data source', () => {
+  it('supports time region query', () => {
+    const frame = doTimeRegionQuery(
+      'test',
+      { fromDayOfWeek: 1, toDayOfWeek: 2 },
+      {
+        from: dateTime('2023-03-01'),
+        to: dateTime('2023-03-31'),
+        raw: {
+          to: '',
+          from: '',
+        },
+      },
+      'utc'
+    );
+
+    expect(toDataFrameDTO(frame!)).toMatchSnapshot(); // inline??
```

Comment:
maybe an inline snapshot - main benefit here seems to be not needing to switch around files to verify correct value is generated by test